docs/en/latest/plugins/zipkin.md

```diff
@@ -27,249 +27,239 @@ description: This document contains information about the Apache zipkin Plugin.
+[Zipkin](https://github.com/openzipkin/zipkin) is an open-source distributed tracing system. The `zipkin` Plugin instruments APISIX and sends traces to Zipkin based on the [Zipkin API specification](https://zipkin.io/pages/instrumenting.html).
 
-## Attributes
+The Plugin can also send traces to other compatible collectors, such as [Jaeger](https://www.jaegertracing.io/docs/1.51/getting-started/#migrating-from-zipkin) and [Apache SkyWalking](https://skywalking.apache.org/docs/main/latest/en/setup/backend/zipkin-trace/#zipkin-receiver), both of which support Zipkin [v1](https://zipkin.io/zipkin-api/zipkin-api.yaml) and [v2](https://zipkin.io/zipkin-api/zipkin2-api.yaml) APIs.
 
-| Name         | Type    | Required | Default        | Valid values | Description                                                                     |
-|--------------|---------|----------|----------------|--------------|---------------------------------------------------------------------------------|
-| endpoint     | string  | True     |                |              | Zipkin HTTP endpoint. For example, `http://127.0.0.1:9411/api/v2/spans`.        |
-| sample_ratio | number  | True     |                | [0.00001, 1] | How often to sample the requests. Setting to `1` will sample all requests.      |
-| service_name | string  | False    | "APISIX"       |              | Service name for the Zipkin reporter to be displayed in Zipkin.                 |
-| server_addr  | string  | False    | `$server_addr` |              | IPv4 address for the Zipkin reporter. You can specify your external IP address. |
-| span_version | integer | False    | 2              | [1, 2]       | Version of the span type.                                                       |
+## Static Configurations
 
-Each traced request will create the spans shown below:
-
-```
-request
-├── proxy: from the beginning of the request to the beginning of header filter
-└── response: from the beginning of header filter to the beginning of log
-```
+By default, `zipkin` Plugin NGINX variables configuration is set to false in the [default configuration](https://github.com/apache/apisix/blob/master/apisix/cli/config.lua):
 
-For older versions (set `span_version` attribute to `1`), these spans are created:
+To modify this value, add the updated configuration to `config.yaml`. For example:
 
-```
-request
-├── rewrite
-├── access
-└── proxy
-    └── body_filter
+```yaml
+plugin_attr:
+  zipkin:
+    set_ngx_var: true
 ```
 
-:::note
+Reload APISIX for changes to take effect.
 
-The span name doesn't represent the corresponding Nginx phase.
+## Attributes
 
-:::
+See the configuration file for configuration options available to all Plugins.
 
-### Sample code for upstream configuration
+| Name         | Type    | Required | Default        | Valid values | Description                                                                     |
```

Comment:
I don't think the names need to be in inline code?

<img width="830" alt="image" src="https://github.com/user-attachments/assets/dc463c24-b046-4fe0-80f7-eac76a59d41c" />

Reply:
OK, also fixed the attributes in the previous release prometheus docs